### ISSUE TITLE
Unused exclusion pattern

### ISSUE THREAD

**Issue by maskit** [apache/incubator-pulsar#684]
#### Expected behavior
No errors during build

#### Actual behavior
```
[INFO] --- maven-assembly-plugin:2.4.1:single (make-assembly) @ pulsar-checksum ---
[INFO] Reading assembly descriptor: src/main/assembly/assembly.xml
[WARNING] The following patterns were never triggered in this artifact exclusion filter:
o  'com.scurrilous'
```

#### Steps to reproduce

```
mvn install -DskipTests
```

#### System configuration
**Pulsar version**: master c51eaad
**OS**: macOS 10.12

**Comment by maskit** [comment]
#678 fixes this.

**Comment by merlimat** [comment]
Merged #678
